v2/go.mod

```diff
@@ -2,4 +2,12 @@ module github.com/googleapis/gax-go/v2
 
 go 1.11
 
-require google.golang.org/grpc v1.19.0
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/golang/protobuf v1.5.0
+	github.com/stretchr/testify v1.7.0
```

Comment:
Please do not use third-party dependencies. Remove this and either use github.com/google/go-cmp for comparison or do direct `==` comparisons in `if` statements. Look at the other tests for examples.